Packages/vcs/Lib/configurator.py

```diff
@@ -19,128 +21,61 @@
 
 import copy
 
-def sync_template(src, target):
```

Comment:
No worries -- linters are good! Thanks for pointing out the removed function so we know it was intentional.